theme/tests/multiplatform.py

```diff
@@ -98,6 +97,7 @@ def setUp(self):
                     groups=[group]
```

Comment:
I have not. I'm looking for input on how to setup the groups and perms. What would effect would leaving groups empty have?